<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleDialog</class>
 <widget class="QDialog" name="ScheduleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="plainTextEditText"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblText">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="dateTimeEditExpiration"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblExpiration">
         <property name="text">
          <string>Expiration</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditTitle"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblTitle">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditTitle</tabstop>
  <tabstop>plainTextEditText</tabstop>
  <tabstop>dateTimeEditExpiration</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
